app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PasswordsController < ApplicationController
```

Comment:
Built out into its own controller because we're only working with passwords in this particular branch.